js/employee-form.js

```diff
@@ -0,0 +1,299 @@
+    setElementAttribute('.edit-employee-profile','src', selectedEmp.img)
+}
+  
+function closeEditEmployeeForm() {
+    let elementsToHide = ["employees-container","alphabet-filter","reset-filter",];
+    elementsToHide.forEach((elementClass) => {
+      changeElementDisplay(`.${elementClass}`,'flex')
+    });
+    changeElementDisplay(".edit-employee-form-container",'none')
+    changeElementDisplay(".employee-table-container",'block')
+    localStorage.removeItem("empRow");
+    let form = document.querySelector('.edit-employee-form');
+    validateField(form, false,'none')
+    form.querySelector('#edit-emp-role').innerHTML="";
+}
+
+function validateField(form, flag = true, mode) {
+    const dangerInputName = {
+      "img": "Image",
+      "fname": "First Name",
+      "lname": "Last Name",
+      "email": "Email",
+      "location": "Location",
+      "dept": "Department",
+      "empNo": "Emp Number",
+      "status": "Status",
+      "joinDate": "Join Date",
+      "dob": "Date of Birth",
+      "projectAssigned": "Project",
+      "managerAssigned": "Manager",
+      "mobile": "Mobile Number",
+    }
+    let formInput = form.getElementsByTagName("input");
+    let check = 1;
+    for (let key in formInput) {
+      let element = formInput[key];
+      if (element.name == 'empNo') {
+        let empNo = element.value.toUpperCase()
+        if (empNo == "") {
+          showValidInput(element, `&#9888; ${dangerInputName[element.name]} is required`, flag);
+          check = 0
+        }
+        else if (!empNo.startsWith("TZ")) {
+          showValidInput(element, `&#9888; employee Id Should start with TZ`, flag)
+          check = 0;
+        }
+        else if(!empNo.match(/^TZ[0-9]+$/)){
+          showValidInput(element, `&#9888; ${dangerInputName[element.name]} should have number starting with TZ`, flag);
+          check = 0
+        }
+        else if (empNo.startsWith("TZ") && mode == 'add') {
+          for (let i = 0; i < employeeList.length; i++) {
+            if (employeeList[i].empNo == empNo) {
+              showValidInput(element, `&#9888;This Emp Number is already taken`, flag)
+              check = 0;
+            }
+          }
+        }
+        
+      }
+      else if (element.name == 'number') {
+        let empNum = element.value
+        if (empNum == "") {
+          showValidInput(element, `&#9888; ${dangerInputName[element.name]} is required`, flag);
+          check = 0
+        }
+        else if (empNum.length != 10) {
+          showValidInput(element, `&#9888; Mobile number should be of 10 digit`, true)
+          check = 0;
+        }
+      }
+      else if (element.type == 'email') {
+        let email = element.value.toLowerCase();
+        if (!email.endsWith("tezo.com")) {
+          showValidInput(element, `&#9888;Email should be of tezo`,flag)
+          check = 0;
+        }
+      }
+      if (element.type != "file") {
+        if (element.name != "dob") {
```

Comment:
Try to minimize using if conditions. Try using && or || operators correspondingly.